wordfreq_builder/wordfreq_builder/cli/tokenize_reddit.py

```diff
@@ -0,0 +1,14 @@
+from wordfreq_builder.tokenizers import cld2_reddit_tokenizer, tokenize_by_language
```

Comment:
Alright @rspeer, if you'd rather not address this then I'm ready to merge this PR.  Let me know.